packages/eslint-plugin/tests/rules/no-useless-template-literals.test.ts

```diff
@@ -409,11 +409,16 @@ declare const nested: string, interpolation: string;
   \`le\${  \`ss\`  }\`
 }\`;
       `,
-      output: `
+      output: [
```

Comment:
yeah, as the author of this test case it really bugged me that I had to write this with the first pass only. Very excited to see this feature land!